tests/Agent/IntegrationTests/UnboundedIntegrationTests/MySql/MySqlConnectorTests.cs

```diff
@@ -133,110 +142,675 @@ public void Test()
         }
     }
 
+    /*** ExecuteReader Tests ***/
```

Comment:
You could consider using `#region` blocks to help organize all of the sub-classed tests: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#defining-regions